### PR TITLE
Making My Portfolio

### PR DESCRIPTION
**Pull request by nitishbharti235**
Work of Week 1.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,24 @@
...
+       'All of the disadvantages in this world stems from a person\'s lack of ability',
+       'The world runs on power. Everything is determined by the superior power. You are weak. That is why you lose',
+       'All suffering in the world is born from an individual\'s incompetence',
+       'As long as it\'s for the right cause, there\'s nothing wrong with playing dirty.',
+       'We can only live while we lose'
+   ]   ;
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  //const greeting = greetings[Math.floor(Math.random() * greetings.length)];
```
Comment (pitankar) [inline portfolio/src/main/webapp/script.js:33]: Should this be removed or uncommented??

Comment (pitankar) [reply portfolio/src/main/webapp/script.js:33]: looks like this should be removed.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,24 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+
```
Comment (pitankar) [inline portfolio/src/main/webapp/script.js:19]: thought: Should we remove newline on 19?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,21 @@
...
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Nitish Bharti's Portfolio</h1>
+      <p>
+          I'm Nitish Bharti persuing BTech Cse degree from IIT(ISM) Dhanbad.<br>
```
Comment (pitankar) [inline portfolio/src/main/webapp/index.html:13]: thought: should `Cse` be `CSe`?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -8,11 +8,21 @@
...
+          I like watching anime.<br>
+          tbh I'm just random type person who has no idea what's happening but I cope up with everything
+          and I do things deliberately and honestly with my heart and my energy.<br>
+          Focus and Patient these two things I follow<br><br>
+          <a href="https://www.linkedin.com/in/nitish-bharti/">LinkedIn Profile</a><br>
+          <a href="https://github.com/nitishbharti235">Github Profile</a> <br>
+      </p>
+      <p>This is my portfolio page and here is my faviourite anime character Ken kaneki.</p>
+      <img src="images/kaneki.png"/>
```
Comment (pitankar) [inline portfolio/src/main/webapp/index.html:22]: `images/kaneki.png` - Be mindful of using images. Consider the personality, intent of use and the usage permissions on the image :)

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (nitishbharti235) [reply portfolio/src/main/webapp/script.js:19]: Well, it looks good after removing it.
So I'm removing it :smile:

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (nitishbharti235) [reply portfolio/src/main/webapp/index.html:13]: changing it to CSE :smile_cat:

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (nitishbharti235) [reply portfolio/src/main/webapp/index.html:22]: I checked it is free to use.

Comment (pitankar) [reply portfolio/src/main/webapp/index.html:22]: Sounds good. marking this as resolved.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (pitankar) [reply portfolio/src/main/webapp/script.js]: :D sounds good. Marking this as resolved.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (pitankar) [reply portfolio/src/main/webapp/script.js]: This seems to be resolved in a later commit. Marking as resolved.

**Pull request merged**